src/components/FilterNotes.js

```diff
@@ -1,12 +1,10 @@
 import styled from 'styled-components/macro';
-import { useState } from 'react';
-
-export default function FilterNotes({ filterNotes, notes }) {
-  const [active, setActive] = useState('all');
 
+export default function FilterNotes({ active, setActive, filterNotes, setSearchInput }) {
   function handleOnClick(event) {
     filterNotes(event.target.name);
```

Comment:
The same naming of filterNotes function and FilterNotes component could be a little bit confusing at some point ;)